spec/features/mainstream_browse_pages_spec.rb

```diff
@@ -55,6 +64,45 @@
     click_on('Citizenship in the UK')
     expect(page).to have_content('Voting')
 
+    # And a draft should have been sent to publishing-api (pending)
+    false and assert_publishing_api_put_draft_item('/browse/citizenship', {
+      "title" => "Citizenship in the UK",
+      "description" => "Voting",
+      "format" => "mainstream_browse_page",
+    })
+
+    # And the page should have been updated in Panopticon
+    assert_tag_updated_in_panopticon(
+      :tag_type => 'section',
+      :tag_id => 'citizenship',
+      :title => 'Citizenship in the UK',
+      :description => 'Voting',
```

Comment:
Actually, after further investigation, the Panopticon API has a separate `/publish` endpoint for changing the state of a tag from draft to published.  The create/update endpoints don't change the state.